### PR TITLE
[AIP-51] Add helper to import default executor class

### PR DESCRIPTION
**Pull request by o-nikolas**
This will be useful for the several points during AIP-51 where the default executor class needs to be imported to replace specific points of coupling.
Also add missing testing for executor loader import mechanism

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by o-nikolas** [comment]
@pierrejeambrun 
This would be nice to use for #27941 and future PRs!

**Review by pierrejeambrun — APPROVED** [review]

**Comment by pierrejeambrun** [comment]
Really useful indeed, thanks. I'll update that when this one is merged :)

File: airflow/executors/executor_loader.py [hunk 1]
```diff
@@ -65,18 +65,23 @@ class ExecutorLoader:
         DEBUG_EXECUTOR: "airflow.executors.debug_executor.DebugExecutor",
     }
 
+    @classmethod
+    def get_default_executor_name(cls) -> str:
+        """Returns the default executor name from Airflow configuration
```
Comment (bdsoha) [inline airflow/executors/executor_loader.py]:
```suggestion
        """Return the default executor name from Airflow configuration.
```

Comment (o-nikolas) [reply airflow/executors/executor_loader.py]: If you view the rest of the module, all the docs use the plural verb form (`Loads the executor...`, `Creates a new instance of...`) so I will keep that form here. But I will add the missing period!

**Review by eladkal — APPROVED** [review]
LGTM

**Pull request merged**